examples/models/smollm3/convert_weights.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+from typing import Dict
+
+import torch
+
+from torchtune.models.convert_weights import get_mapped_key
+
+from torchtune.training import FullModelHFCheckpointer
```

Comment:
Can we use this instead? I'd like to move away from TorchTune if possible

https://github.com/pytorch/executorch/pull/13011/files#diff-791e820b8f381c55f72cd81f161de5b514d0b3c7139f4dda3fcbe5a2522f2299R1